environment.yml

```diff
@@ -4,34 +4,33 @@
 name: svgdigitizer-build
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - astropy>=5,<6
-  - black
-  - click=8.0.3
+  - black>=23,<24
+  - click>=8,<9
   - frictionless>=5.8.3,<6
   - isort
   - jupytext
   - matplotlib-base>=3.5,<4
   - mergedeep>=1.3.4,<2
-  - myst-parser>=0.15,<0.19.0
-  - myst-nb>=0.13.0,<0.18.0
+  - myst-parser
+  - myst-nb
   - pandas>=1.3,<2
   - pdf2image>=1.16,<2
   - pillow>=10.0.1,<11
-  - pip>=21,<22
-  - poppler>=22,<23
+  - pip
+  - poppler
   - pybtex>=0.24,<0.25
-  - pylint>=2.12,<3
-  - pytest>=6.2,<7
-  - python>=3.9,<4
+  - pylint>=3,<3.1
```

Comment:
This should probably also not have a version number?

Reply:
Same as above. Pylint changes its rules when the version jumps. That can be annoying because when a new pylint gets released, PRs are going to fail because of that version change because the rules changed in pylint.